core-blocks/gallery/sortable.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { SortableContainer, SortableElement, arrayMove } from 'react-sortable-hoc';
+
+const SortableItem = SortableElement( ( { children } ) => children );
+
+class Sortable extends Component {
+	//constructor
```

Comment:
Why comment here?

Reply:
Also, we could get rid of the constructor all together using by using class properties.

Reply:
You are right let me fix that

Reply:
we don't have support for class properties enabled though, and other blocks/components seem to be relying on binding inside the constructor as well. Should we enable it?

Reply:
We're reluctant to use new language features until they've reached stage 4. [Class fields are still at stage 3](https://github.com/tc39/proposal-class-fields).